### ISSUE TITLE
Support IEA in Blueprints

### ISSUE THREAD

**Issue by jessepeterson** [micromdm/micromdm#486]
Currently we support `InstallEnterpriseApplication` via the API. But by default we send an `InstallApplication` command via initial profiles. Support sending IEA and supplying the pinned certs in  enrollment blueprints.

**Comment by jessepeterson** [comment]
Note we may want to be careful about how this is used. It will only be available to >= 10.13.6 macOS hosts.

**Comment by erikng** [comment]
If you make logic for this (OS deployment) this code could also be used for profiles, especially things like Kernel Extensions where if you install the profile prior to 10.13 and then upgrade, the profile won't work.

Thanks,
Erik Gomez
________________________________

**Comment by jessepeterson** [comment]
Yeah that sort of logic I don't think will ever be in the 'core'/platform MicroMDM offering. By design it doesn't want to get into the business of managing that sort of (almost) implementation detail. That's why we have webhooks & APIs so other components/tools can do that work.
